src/main/java/teammates/storage/api/FeedbackResponsesDb.java

```diff
@@ -43,6 +43,35 @@ public void createFeedbackResponses(Collection<FeedbackResponseAttributes> respo
      * @return Null if not found.
      */
     public FeedbackResponseAttributes getFeedbackResponse(String feedbackResponseId) {
+        FeedbackResponse feedbackResponse = getFeedbackResponseEntityOptimized(feedbackResponseId);
+        if (feedbackResponse == null) {
+            return null;
+        }
+        return new FeedbackResponseAttributes(feedbackResponse);    
+    }
+
+    /**
+     * Preconditions: <br>
+     * * All parameters are non-null.
+     * @return Null if not found.
+     */
+    public FeedbackResponseAttributes getFeedbackResponse (
+            String feedbackQuestionId, String giverEmail, String receiverEmail) {
+        FeedbackResponse feedbackResponse = 
+                getFeedbackResponseEntityOptimized (feedbackQuestionId, giverEmail, receiverEmail);
+        if (feedbackResponse == null) {
+            return null;
+        }
+        return new FeedbackResponseAttributes(feedbackResponse);        
+    }
+    
+
+    /**
+     * Preconditions: <br>
+     * * All parameters are non-null. 
+     * @return Null if not found.
+     */
```

Comment:
It is... not optimized. It is a public version of getFeedbackResponseEntity.
Renamed to getFeedbackResponseEntityWithCheck.